### PR TITLE
Do not revert signature fraud submission TX if slashing failed

### PR DESCRIPTION
**Pull request by pdyraga**
Closes: https://github.com/keep-network/keep-ecdsa/issues/356

In practice, the situation when KEEP token slashing failed should never happen (see https://github.com/keep-network/keep-core/pull/1625). However, we want to be very explicit in this contract and protect the owner that even if it happens, the transaction submitting fraud proof is not going to fail and keep owner can seize and liquidate bonds in the same transaction.

### CODE REVIEW

File: solidity/test/BondedECDSAKeepTest.js [hunk 1]
```diff
@@ -1175,6 +1189,8 @@ contract("BondedECDSAKeep", (accounts) => {
         "Caller is not the keep member"
       )
     })
+
+    it("does not fail if slashing failed", async () => {})
```
Comment (nkuba) [inline solidity/test/BondedECDSAKeepTest.js:1193]: Missing implementation?

File: solidity/test/BondedECDSAKeepTest.js [hunk 2]
```diff
@@ -986,46 +986,60 @@ contract("BondedECDSAKeep", (accounts) => {
...
+          signature1.R,
+          signature1.S,
+          hash256Digest1,
+          preimage1
+        ),
+        "Keep is not active"
+      )
     })
 
-    it("slashes keep members stakes", async () => {
```
Comment (nkuba) [inline solidity/test/BondedECDSAKeepTest.js:1012]: Are we removing this test on purpose?

File: solidity/test/BondedECDSAKeepTest.js (hunk 1 above)
Comment (pdyraga) [reply solidity/test/BondedECDSAKeepTest.js]: I had problems with linter, I do not know what happens here. I removed this line. Test for this scenario is implemented [here](https://github.com/keep-network/keep-ecdsa/blob/77b95f9e47b908dc30503e315aa04e80b2f0c8ce/solidity/test/BondedECDSAKeepTest.js#L1020-L1042).

File: solidity/test/BondedECDSAKeepTest.js (hunk 2 above)
Comment (pdyraga) [reply solidity/test/BondedECDSAKeepTest.js:1012]: Yes, this is not covered [here](https://github.com/keep-network/keep-ecdsa/blob/77b95f9e47b908dc30503e315aa04e80b2f0c8ce/solidity/test/BondedECDSAKeepTest.js#L932-L959).

**Pull request merged**